wp-content/plugins/core/src/Theme/Branding/Login_Screen.php

```diff
@@ -27,7 +27,7 @@ public function inject_login_logo(): void {
 		$logo_height = $logo_data[2] / 2;
 
 		?>
-		<style type="text/css">
+		<style>
```

Comment:
Why remove the type?

Reply:
phpstorm told me it is deprecated :man_shrugging:

Reply:
#TIL https://developer.mozilla.org/en-US/docs/Web/API/HTMLStyleElement/type